contrib/lkt/language/parser.py

```diff
@@ -1246,6 +1295,11 @@ def type_base_scope():
 
     call_scope = Property(Entity.type_scope)
 
+    is_generic = Property(
+        Entity.is_a(T.InstantiatedGenericType), public=True,
+        doc="Returns whether this type is an instantiated generic type"
```

Comment:
This should be “return”, and this lacks a trailing dot.

Reply:
As discussed live, I have no intention of following PEP 257 here or elsewhere, and I'll add the trailing dot :slightly_smiling_face: